### DISCUSSION TITLE
[Q] how can I use write-good or cspell on all file type

### DISCUSSION THREAD

**Discussion by arthurkiller** [w0rp/ale#4655] [Q&A - Ask for help with problems]
<img width="929" alt="image" src="https://github.com/dense-analysis/ale/assets/11133870/c18b005b-ae98-4c35-b034-944f25e81cb4">

**Comment by arthurkiller** [comment]
I have read this
https://github.com/dense-analysis/ale/discussions/4037

Does that mean we should support this ourselves?

**Reply by hsanson** [reply]
Maybe `g:ale_linter_aliases` can do what you need. Use help to read about it:  `:h g:ale_linter_aliases`.
